packages/tools-iottwinmaker/src/commands/deploy.ts

```diff
@@ -51,17 +52,24 @@ export const builder: CommandBuilder<Options> = (yargs) =>
       require: true,
       description: 'Specify the project location, directory for tmdt.json file',
     },
+    'endpoint-url': {
+      type: 'string',
+      require: false,
+      description: 'Specify the AWS IoT TwinMaker endpoint.',
+      default: '',
+    },
   });
 
 export const handler = async (argv: Arguments<Options>) => {
   const workspaceId: string = argv['workspace-id']; // TODO allow it to be optional (i.e. option to autogenerate workspace for them)
   const region: string = argv.region;
   const dir: string = argv.dir;
+  const tmEndpoint: string = argv['endpoint-url'];
   console.log(
     `Deploying project from directory ${dir} into workspace ${workspaceId} in ${region}`
   );
 
-  initDefaultAwsClients({ region: region });
+  initDefaultAwsClients({ region, tmEndpoint });
```

Comment:
strict types here? e.g. (region: region, tmEndpoint: string)

Reply:
`region` isn't a type, it's the name of the variable and the name of the input field. In Typescript you don't need to specify the value of a named input if the name is the same.

i.e. `{ region: region }` is equivalent and inferred to be `{ region }`